dl1_data_handler/reader.py

```diff
@@ -242,6 +245,13 @@ class DLDataReader(Component):
         ),
     ).tag(config=True)
 
+    quality_cuts = Bool(
+        default_value=True,
+        help=(
+            "Require quality cuts by ``TableQualityQuery``. False for nsb trigger."
+        ),
+    ).tag(config=True)
+
```

Comment:
`TableQualityQuery` should be empty if you do not want to pass any quality cuts. I know it is a quite convenient shortcut for you, but I would prefer to have properly handled by the configuration system. We should write example configs for the different use cases.